uPortal-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -90,15 +90,15 @@
         <filter-class>org.jasig.cas.client.validation.Cas20ProxyReceivingTicketValidationFilter</filter-class>
         <init-param>
           <param-name>casServerUrlPrefix</param-name>
-          <param-value>${environment.build.cas.protocol}://${environment.build.cas.server}${environment.build.cas.context}</param-value>
+          <param-value>http://localhost:8080/cas</param-value>
```

Comment:
Yeah, this item is tougher than other items.

I included this change because (ATM) this value is _literally_ hard-coded to...

```
${environment.build.cas.protocol}://${environment.build.cas.server}${environment.build.cas.context}
```

There are no more Maven Filters, so the replacement tokens end up in the deployment.  :(  That's why the 'Sign In' link is broken on `master` now.

At the present moment, you can (100%) configure this setting to a local value by overlaying uPortal's `web.xml` file in uPortal-start.  That works, but I hope to do better (just not in this PR.)

The reading I've done suggests that we can (ultimately) avoid overlaying `web.xml` using JVM args (but not environment variable):  https://stackoverflow.com/questions/1404384/referencing-environment-variables-in-web-xml